src/components/Incident/IncidentSummary.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import SyncIcon from 'material-ui/svg-icons/notification/sync'
+import IconButtonStyled from '../elements/IconButtonStyled'
+import ModeEditIcon from 'material-ui/svg-icons/editor/mode-edit'
+import Engagements from '../Engagements'
+import GlobalActions from '../Timeline/Playbook/GlobalActions'
+
+export const IncidentSummaryName = (ticketId) => {
+    return 'IncidentSummary' + (ticketId ? '_' + ticketId : '')
+}
+
+export const IncidentSummary = (incident, ticket, ticketSystem, ticketOriginId, dispatch) =>
+[
+    HeaderRow(ticketOriginId),
+    TicketDetailsRow(ticketSystem, ticket),
+    TitleRow(incident),
+    GlobalActionsRow(incident, ticketOriginId),
+    EngagementsRow(incident, dispatch)
+]
+
+const HeaderRow = (ticketId) => [
+    [
+        (key) =>
+            <strong key={key}>
+                Incident Summary{ticketId ? ` for ${ticketId}`:''}:
+                &nbsp;
+                <IconButtonStyled tooltip="Refresh">
+                    <SyncIcon />
+                </IconButtonStyled>
+            </strong>
+    ]
+]
+
+const TicketDetailsRow = (ticketSystem, ticket) => [
+    [
+        (key) =>
```

Comment:
Please name these functions so that the last line of the `TicketDetailsRow` function can be easier to read as `[[foo, bar], [baz, quux]]`

Reply:
The difficult part here is the closures. It would have to be something along the lines of [[foo(ticket, ticketSystem), bar(ticket)],[baz, quux]]

I broke it up into columns. I'd rather put more effort into finding a way to do it without the (key) => portion before going down that route.